package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "build": "babel src --out-dir lib",
     "prepublish": "rimraf lib && npm run build",
-    "test": "mocha --compilers js:babel-core/register --reporter spec test/*.js"
+    "test": "mocha --compilers js:babel-core/register --reporter spec test/*.js",
+    "lint": "eslint src test"
```

Comment:
@TrySound 
Yeah and `posttest` hook would simplify things I suppose. 

So maybe something like this?

```js
// package.json

  "scripts": {
    "build": "babel src --out-dir lib",
    "prepublish": "npm run test && rimraf lib && npm run build",
    "test": "mocha --compilers js:babel-core/register --reporter spec test/*.js",
    "posttest": "npm run lint",
    "lint": "eslint src test"
  }
```